src/components/ReportActionAvatars/useReportActionAvatars.ts

```diff
@@ -123,12 +124,13 @@ function useReportActionAvatars({
 
     const avatarsForAccountIDs: IconType[] = accountIDsToMap.map((id) => {
         const invitedEmail = invitedEmailsToAccountIDs ? Object.keys(invitedEmailsToAccountIDs).find((email) => invitedEmailsToAccountIDs[email] === id) : undefined;
+        const formattedInvitedLogin = invitedEmail ? addSMSDomainIfPhoneNumber(invitedEmail) : undefined;
```

Comment:
```suggestion
        const formattedInvitedLogin = addSMSDomainIfPhoneNumber(invitedEmail ?? '');
```

And we could even do it inline below: 

```
getDefaultAvatar({accountID: id, accountEmail: addSMSDomainIfPhoneNumber(invitedEmail ?? '')}) : undefined,
```